server/tests/test.py

```diff
@@ -0,0 +1,120 @@
+import base64
```

Comment:
Follow conventions. @noobed-max assist @Ashish9738 here.

Reply:
Modified the changes.